libraries/radpdfprocessing/cross-platform.md

```diff
@@ -27,29 +27,22 @@ To export images different than Jpeg and Jpeg2000 or ImageQuality different than
 * **Telerik.Documents.ImageUtils.dll**
 <br><sub>_This assembly is not available in UI for Xamarin._</sub>
 
->note The **Telerik.Documents.ImageUtils.dll** assembly depends on **ImageSharp** and **TiffLibrary.ImageSharpAdapter**. In order to use this assembly, you will need to add references to [ImageSharp](https://www.nuget.org/packages/SixLabors.ImageSharp/) and [TiffLibrary.ImageSharpAdapter](https://www.nuget.org/packages/TiffLibrary.ImageSharpAdapter/).
-
 > Note that for .NET Framework & .NET Core with Windows Compatibility Pack projects, the references contain "Windows" in their names (e.g. **Telerik.Windows.Documents.Core.dll**)
 
-## What's New
-
-In the .NET Framework version of PdfProcessing, scenarios like reading fonts and converting images or scaling their quality are something that comes out of the box. However, the .NET Standard specification doesn't specify APIs to provide such functionalities built in the library, so there are some differences in both versions of PdfProcessing described in this article.
-
-### New APIs
+## Fonts and Images
 
-The [limitations in .Net Standard](#limitations-in-net-standard) require some additional settings to be done, therefore, the **RadPdfProcessing** library exposes a FixedExtensibilityManager class used for providing extensibility mechanisms.
+The .NET Framework version of PdfProcessing comes with out-of-the-box functionality to read fonts, convert images, and scale their quality. The .NET Standard specification, however does not specify APIs to provide such functionalities built in the library.
+In order to provide the necessary extensibility mechanisms for working with fonts and images, the .NET Standard version of **RadPdfProcessing** exposes the **FixedExtensibilityManager** class.
 
-#### FixedExtensibilityManager class
+### Setting and Exporting Fonts
 
-The new **FixedExtensibilityManager** class is exposing the following properties:
+PdfProcessing needs to have access to the font data so that it can read it and add it to the PDF file. That is why, to allow the library to create and use fonts, you will need to provide an implementation of the FontsProviderBase abstract class and set this implementation to the <u>FontsProvider</u> property of the **FixedExtensibilityManager**.
 
-* **FontsProvider**: Gets or sets a *FontsProviderBase* instance used to provide missing fonts.
+The code snippets below show how to create a custom <u>FontsProvider</u> implementation and how to set it.
```

Comment:
show**S**

Reply:
The code snippets show is correct as snippets is plural.